lib/enkf/tests/enkf_analysis_config.cpp

```diff
@@ -70,9 +70,16 @@ void test_min_realizations(const char * num_realizations_str, const char * min_r
       analysis_config_init(ac, content);
 
       int num_realizations = config_content_get_value_as_int(content, NUM_REALIZATIONS_KEY);
+
       test_assert_false(analysis_config_have_enough_realisations(ac, min_realizations_expected_needed - 1, num_realizations ));
       test_assert_true(analysis_config_have_enough_realisations(ac, min_realizations_expected_needed, num_realizations ));
       test_assert_true(analysis_config_have_enough_realisations(ac, min_realizations_expected_needed + 1, num_realizations ));
+
+      int min_realizations = analysis_config_get_min_realisations(ac);
+      if(min_realizations > 0)
```

Comment:
This is a test, I presume you should know what's the value of `min_realizations` is.

Also, do we have tests to cover both the cases `min_realizations > ensemble_size` and `min_realizations < ensemble_size`?

Reply:
Yes, this test was to for min_realizations > ensemble_size. Tests for other cases were already included.